File: python/scan/env.py
import os
from dotenv import load_dotenv

load_dotenv()

DEBUG = os.getenv('DEBUG')
LND_GRPC_URL = os.getenv('LND_GRPC_URL')
LND_MACAROON = os.getenv('LND_MACAROON')
LND_TLS_CERT = os.getenv('LND_TLS_CERT')

File: node/env.ts
import dotenv from 'dotenv';
dotenv.config();

const env = {
  LND_GRPC_URL: process.env.LND_GRPC_URL as string,
  LND_MACAROON: process.env.LND_MACAROON as string,
  LND_TLS_CERT: process.env.LND_TLS_CERT as string,
};

// Ensure all keys exist
Object.entries(env).forEach(([key, value]) => {
  if (!value) {
    throw new Error(`Required environment variable '${key}' is missing!`);
  }
});

export default env;